app/Http/Controllers/Consumables/ConsumableCheckoutController.php

```diff
@@ -75,13 +81,14 @@ public function store(Request $request, $consumableId)
         // Update the consumable data
         $consumable->assigned_to = e($request->input('assigned_to'));
 
+        for($i = 0; $i < $quantity; $i++){
         $consumable->users()->attach($consumable->id, [
             'consumable_id' => $consumable->id,
             'user_id' => $admin_user->id,
             'assigned_to' => e($request->input('assigned_to')),
             'note' => $request->input('note'),
         ]);
-
+        }
         event(new CheckoutableCheckedOut($consumable, $user, Auth::user(), $request->input('note')));
```

Comment:
We're not passing qty here here in the event though - it will show a checkout, but not the amount that was checked out.